public/app/common/components/cc-form/style.scss

```diff
@@ -0,0 +1,42 @@
+@import "theme";
+
+.cc-form {
+  @include form-row-base();
+
+  /* fix for generated fieldset from form lib */
```

Comment:
Also what is exactly fixed. In case the originating error is later fixed, this fix can then easily be removed